.changeset/early-falcons-return.md

```diff
@@ -0,0 +1,5 @@
+---
+'@scaleway/ui': patch
```

Comment:
It's just a documentation change so it can be a minor

Reply:
Not needed as we discuss this morning ;)

Reply:
@ModuloM to be removed :D